packages/api-reference/src/components/Sidebar/Sidebar.vue

```diff
@@ -183,6 +183,8 @@ onMounted(() => {
 <style scoped>
 .sidebar {
   --scalar-sidebar-indent-base: 12px;
+  --scalar-sidebar-font-weight-active: var(--scalar-semibold);
```

Comment:
We don't want to define these in `variables.css`? We already have a few other sidebar variables in there 

https://github.com/scalar/scalar/blob/main/packages/themes/src/variables.css#L87-L89